lib/instance.js

```diff
@@ -634,10 +634,15 @@ Instance.prototype.save = function(options) {
         var instance = self.get(include.as);
         if (!instance) return Promise.resolve();
 
-        return instance.save({
+        var includeOptions = _.defaults(include.options || {}, {
           transaction: options.transaction,
           logging: options.logging
-        }).then(function () {
+        });
+        _.assign(includeOptions, {
+          parentRecord: self
+        });
```

Comment:
And we should likely also inherit options from the overall create call to support plugins etc